### PR TITLE
(Minor) Style changes to run_chunk

### PR DESCRIPTION
**Pull request by jshoe**
- Mostly style changes for run_chunk() and also some auto-formatting.
- Test run for this: https://staging.idseq.net/samples/588
- I'll bump the version number before merging

### CODE REVIEW

File: idseq_pipeline/commands/non_host_alignment_functions.py [hunk 1]
```diff
@@ -798,53 +804,64 @@ def run_chunk(part_suffix, remote_home_dir, remote_index_dir, remote_work_dir,
         remote_input_files=" ".join(
             remote_work_dir + "/" + input_fa for input_fa in input_files),
         multihit_remote_outfile=multihit_remote_outfile
-        if service == "gsnap" else
-        multihit_remote_outfile[:
-                                -3]  # strip the .m8 for rapsearch as it adds that
+        if service == "gsnap" else multihit_remote_outfile[:-3]
```
Comment (jameshowardwang) [inline idseq_pipeline/commands/non_host_alignment_functions.py]: should this be indented?

File: idseq_pipeline/commands/non_host_alignment_functions.py [hunk 2]
```diff
@@ -798,53 +804,64 @@ def run_chunk(part_suffix, remote_home_dir, remote_index_dir, remote_work_dir,
...
-                    multihit_remote_outfile, multihit_local_outfile))
-            execute_command("aws s3 cp --quiet %s %s/" %
-                            (multihit_local_outfile,
-                             SAMPLE_S3_OUTPUT_CHUNKS_PATH))
-        write_to_log("finished alignment for chunk %s on %s server %s" %
+
+    if lazy_run and fetch_lazy_result(multihit_s3_outfile,
+                                      multihit_local_outfile):
+        # Skip if we already have previously computed results
+        return multihit_local_outfile
```
Comment (jameshowardwang) [inline idseq_pipeline/commands/non_host_alignment_functions.py]: not sure if i agree with the early return

File: idseq_pipeline/commands/non_host_alignment_functions.py [hunk 3]
```diff
@@ -798,53 +804,64 @@ def run_chunk(part_suffix, remote_home_dir, remote_index_dir, remote_work_dir,
...
+            max_concurrent = RAPSEARCH2_MAX_CONCURRENT
+
+        instance_ip = wait_for_server_ip(service, key_path, remote_username,
+                                         ENVIRONMENT, max_concurrent, chunk_id)
+        write_to_log("starting alignment for chunk %s on %s server %s" %
                      (chunk_id, service, instance_ip))
+        execute_command(
+            remote_command(commands, key_path, remote_username, instance_ip))
+
+        # Check if every row has correct number of columns (12) in the output
```
Comment (jameshowardwang) [inline idseq_pipeline/commands/non_host_alignment_functions.py]: is this the right location for this comment? would it be better above the while loop?

File: idseq_pipeline/commands/non_host_alignment_functions.py (hunk 1 above)
Comment (jshoe) [reply idseq_pipeline/commands/non_host_alignment_functions.py]: changed

File: idseq_pipeline/commands/non_host_alignment_functions.py (hunk 2 above)
Comment (jshoe) [reply idseq_pipeline/commands/non_host_alignment_functions.py]: changed

File: idseq_pipeline/commands/non_host_alignment_functions.py (hunk 3 above)
Comment (jshoe) [reply idseq_pipeline/commands/non_host_alignment_functions.py]: changed

**Pull request merged**